### PR TITLE
 add the possibility to run mimir in docker

### PR DESCRIPTION
**Pull request by antoine-de**
You can use docker-compose to run mimir.

The easiest way to do this is:

`pipenv run inv -f docker_settings.yaml load-in-docker-and-test`

It's still a bit rough, but I think it can help ease the geocoder tester run

### CODE REVIEW

File: Pipfile.lock [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
             "os_name": "posix",
             "platform_machine": "x86_64",
             "platform_python_implementation": "CPython",
             "platform_release": "4.13.0-43-generic",
             "platform_system": "Linux",
             "platform_version": "#48-Ubuntu SMP Wed May 16 12:18:48 UTC 2018",
-            "python_full_version": "3.6.3",
-            "python_version": "3.6",
-            "sys_platform": "linux"
+            "python_full_version": "2.7.14",
```
Comment (amatissart) [inline Pipfile.lock]: That's probably not what we want

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+../docker-compose.yml
```
Comment (amatissart) [inline docker-compose.yml]: I am not sure to understand what's going on here.

File: tasks.py [hunk 3]
```diff
@@ -126,3 +131,65 @@ def load_all(ctx):
...
+    if not ctx.run_on_docker_compose:
+        raise Exception("geocoder-tester can run only in docker-compose mode")
+    logging.info('running geocoder-tester')
+
+    files_args = _build_docker_files_args(['tester_docker-compose.yml'] + files)
+
+    ctx.run('docker-compose {files} run --rm geocoder-tester'.format(files=files_args))
+
+
+@retry(stop_max_delay=20000, wait_fixed=10)
```
Comment (amatissart) [inline tasks.py]: 10ms between retries is maybe a bit too short ?

File: tester_docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  geocoder-tester:
+    image: geocoder-tester
```
Comment (amatissart) [inline tester_docker-compose.yml]: Where does this image come from ?

File: docker-compose.yml (hunk 2 above)
Comment (amatissart) [reply docker-compose.yml]: Hum ok, it's a symbolic link

File: readme.md [hunk 5]
```diff
@@ -1,5 +1,52 @@
...
+
+The easiest way to do this is:
+
+`pipenv run inv -f docker_settings.yaml load-in-docker-and-test`
+
+This will:
+
+* run a docker-compose up
+* import some data in mimir
+* run geocoder in it and write results in a local directory
```
Comment (nlehuby) [inline readme.md]: Did you mean `geocoder-tester` ?

**Review by amatissart — APPROVED** [review]

**Pull request merged**